src/main/resources/db/changelog/changes/07-insert-and-assign-roles.yaml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
```

Comment:
Maybe it`s better to divide those change sets, because the alter different tables